demosplan/DemosPlanDocumentBundle/Logic/SingleDocumentService.php

```diff
@@ -476,7 +478,7 @@ public function getNotVisibleSingleDocuments(array $singleDocumentIds, string $p
      *
      * @throws Exception
      */
-    public function createSingleDocumentVersion(SingleDocument $singleDocument): SingleDocumentVersion
+    public function createSingleDocumentVersion(SingleDocumentInterface $singleDocument)
```

Comment:
we did remove it for this case by purpose, the SingleDocumentServiceInterface (where the method createSingleDocumentVersion is defined ) is in demosplan-addon, and the SingleDocumentVersion is not available there. To avoid creating and implementing an new interface ( in this case it would be SingleDocumentVersionInterface ) which is only needed (for now) as return type, we decided to remove it, but u are right we can add the return type in the method annotations

Reply:
https://github.com/demos-europe/demosplan-core/pull/864/commits/4c2ef0ef04c7d413bdab4567e5eec4e7bbed25c0